### PR TITLE
Fixes to layerset, quickstart, sketch and sources pages

### PR DESCRIPTION
**Pull request by rschwammborn**
Small PR with URL path, wording and screenshot updates ("quality control").
Fixes some errors that went into main branch via the latest merges (#365-#368).

### CODE REVIEW

File: en/functions/backend/layerset.rst [hunk 1]
```diff
@@ -145,7 +145,7 @@ Basesources
 
 There are many ways to fill the Layertree and work with basic services:
 - e.g. by hiding them in the layer tree and using the `BaseSourceSwitcher <../basic/basesourceswitcher>`_.
-- or by work with the possibilities of the `thematic layer tree <../basic/layerertree>`_, to create different layersets and distribute the basic services and thematic services in them to place them in the layer tree.
+- or by work with the possibilities of the `thematic layer tree <../basic/layertree>`_, to create different layersets and distribute the basic services and thematic services in them to place them in the layer tree.
```
Comment (astroidex) [inline en/functions/backend/layerset.rst:148]: better: 
- or by work with the possibilities of the `thematic layer tree <../basic/layertree>`_. This option will show the layerset name with a folder and checkbox in the layertree. You can define whether the layerset folder should be shown and whether it should be opened on start.

File: en/functions/backend/layerset.rst [hunk 2]
```diff
@@ -167,7 +167,7 @@ There are also some WMS services that only support a maximum image size that can
 Vendor Specific Parameter
 -------------------------
 
-In a layererset instance, vendor specific parameters are appended to the WMS request. The implementation follows the specifications of the multi-dimensional data in the WMS specification. In Mapbender, the vendor specific parameters can be used to append user and group information of the logged-in user to the WMS request. Fixed values ​​can also be transmitted. The following example shows the definition of a parameter "group", which passes on the group of the user currently logged into Mapbender.
+In a layerset instance, vendor specific parameters are appended to the WMS request. The implementation follows the specifications of the multi-dimensional data in the WMS specification. In Mapbender, the vendor specific parameters can be used to append user and group information of the logged-in user to the WMS request. Fixed values ​​can also be transmitted. The following example shows the definition of a parameter "group", which passes on the group of the user currently logged into Mapbender.
```
Comment (astroidex) [inline en/functions/backend/layerset.rst:170]: In a layerset instance -> In a service instance (NOT layerset instance)

Comment (astroidex) [inline en/functions/backend/layerset.rst:170]: Shall we delete the sentence? I do not get the meaning.
The implementation follows the specifications of the multi-dimensional data in the WMS specification.

File: en/functions/backend/layerset.rst [hunk 3]
```diff
@@ -186,7 +186,7 @@ Further information
 
 * Further tips on manipulating WMS can be found in the `FAQs <../../faq#performance>`_.
 
-* The relevance of layersets for the display in the layertree is described in the Thematic Layers section of the `layertree documentation <../basic/layertree>`_
+* The relevance of layersets for the display in the layertree is described in the Thematic Layers section of the `layertree documentation <../basic/layertree#workflow-thematic-layertree>`_.
 
 * Layersets can be switched on or off for a map completely in the `Map element <../basic/map>`_ settings.
```
Comment (astroidex) [inline en/functions/backend/layerset.rst:191]: Layersets can be switched on or off for a map completely
--> better: Layersets be disabled for the application in the `Map element <../basic/map>`_ settings.

File: en/quickstart.rst [hunk 4]
```diff
@@ -152,12 +152,13 @@ Furthermore, new applications can be created from scratch. The required steps ar
 
 #. Click *"save"* to save and create your application. It is now possible to add elements (e.g. map, navigation bar, legend) and services to your applicaiton.
 
-Applications can be re-edited at any point. In order to do so, navigate to the application overview. Here, you can click on a |mapbender-button-edit| ``Button`` for every application with corresponding editing rights. Furthermore, applications can be viewed in the Frontend via a |mapbender-button-show| Button. They can also be exported (|mapbender-button-export|), deleted (|mapbender-button-delete|) or give public access (|mapbender-button-publish|). 
-
   .. image:: ../figures/mapbender_create_application.png
      :width: 100%
 
 
+Applications can be re-edited at any point. In order to do so, navigate to the application overview. Here, you can click on a |mapbender-button-edit| ``Button`` for every application with corresponding editing rights. Furthermore, applications can be viewed in the Frontend via a |mapbender-button-show| Button. They can also be exported (|mapbender-button-export|), deleted (|mapbender-button-delete|) or give public access (|mapbender-button-publish|).
```
Comment (astroidex) [inline en/quickstart.rst:159]: maybe delete at any point.

**Review by astroidex — CHANGES_REQUESTED** [review]

**Comment by rschwammborn** [comment]
Closing this because I didn't check out a clean new branch while working on #371. The mentioned PR carries all changes made here (+ edits from change requests) and can be used as replacement.

**Pull request closed without merging**